### PR TITLE
Stability cases: stop kubelet and etcd

### PR DESCRIPTION
**Pull request by weekface**
### What problem does this PR solve? 

- stop all kubelets
- stop all etcds
- data security after deleting the database

### Check List 

Tests 

 - Unit test
 - E2E test
 - Stability test

Code changes

 - Has Go code change

Side effects

Related changes

 - Need to cherry-pick to the release branch

### Does this PR introduce a user-facing change?:
 
 ```release-note
NONE
 ```

### CODE REVIEW

File: tests/failover.go [hunk 1]
```diff
@@ -173,7 +173,7 @@ func (oa *operatorActions) CheckFailoverPending(info *TidbClusterConfig, node st
 				if _, exist := affectedPods[failureStore.PodName]; exist {
 					err := fmt.Errorf("cluster: [%s] the tikv store[%s] should be mark failure after %s", info.FullName(), failureStore.PodName, deadline.Format(time.RFC3339))
 					glog.Errorf(err.Error())
-					return false, err
+					return false, nil
```
Comment (weekface) [inline tests/failover.go]: There may have been a failover before.

File: tests/actions.go [hunk 2]
```diff
@@ -950,7 +1020,8 @@ func (oa *operatorActions) CheckUpgrade(ctx context.Context, info *TidbClusterCo
 				}
 				glog.V(4).Infof("index:%d,schedulers:%v,error:%v", i, schedulers, err)
 				if len(schedulers) > 1 {
-					return true, fmt.Errorf("there are too many evict leader schedulers: %v", schedulers)
+					glog.Errorf("there are too many evict leader schedulers: %v", schedulers)
+					return false, nil
```
Comment (weekface) [inline tests/actions.go]: There may be two schedulers at the same time

File: tests/failover.go (hunk 1 above)
Comment (gregwebs) [reply tests/failover.go]: should this be a comment in the code?

**Pull request merged**